portable/ARMv8M/non_secure/port.c

```diff
@@ -568,34 +579,61 @@ PRIVILEGED_DATA static volatile uint32_t ulCriticalNesting = 0xaaaaaaaaUL;
                 portNVIC_SYSTICK_LOAD_REG = ( ( ulCompleteTickPeriods + 1UL ) * ulTimerCountsForOneTick ) - ulCompletedSysTickDecrements;
             }
 
-            /* Restart SysTick so it runs from portNVIC_SYSTICK_LOAD_REG again,
+            /*
+             * Restart SysTick so it runs from portNVIC_SYSTICK_LOAD_REG again,
              * then set portNVIC_SYSTICK_LOAD_REG back to its standard value.  If
              * the SysTick is not using the core clock, temporarily configure it to
-             * use the core clock.  This configuration forces the SysTick to load
+             * use the core clock. This configuration forces the SysTick to load
              * from portNVIC_SYSTICK_LOAD_REG immediately instead of at the next
-             * cycle of the other clock.  Then portNVIC_SYSTICK_LOAD_REG is ready
-             * to receive the standard value immediately. */
+             * cycle of the external clock. Then portNVIC_SYSTICK_LOAD_REG is ready
```

Comment:
I belive the other clock can also be internal - if a peripheral time is considered internal - but I may be wrong.

Reply:
would you prefer the term "implementation-defined clock?"

Reply:
The ARM documentation refers to this as the "Uses the IMPLEMENTATION DEFINED external reference clock."